mpitests/test_adjoint_jacobian.py

```diff
@@ -188,7 +188,7 @@ def test_pauli_rotation_gradient(self, G, theta, tol, isBatch_obs, request):
         dev_cpu = qml.device("default.qubit", wires=3)
 
         with qml.tape.QuantumTape() as tape:
-            qml.QubitStateVector(np.array([1.0, -1.0]) / np.sqrt(2), wires=0)
+            qml.StatePrep(np.array([1.0, -1.0]) / np.sqrt(2), wires=0)
```

Comment:
Here we could parametrize the test over the state prep op since `QubitStateVector` should still be supported. If we go forward with #134, I will do that there.